### PR TITLE
Manage sources

### PR DESCRIPTION
**Pull request by mconnell**
Adds the ability to add/edit sources

### CODE REVIEW

File: app/views/admin/sources/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<h1>Sources</h1>
+
+<%= link_to 'Add Source', new_admin_source_path %>
+
+<table>
```
Comment (benbarnett) [inline app/views/admin/sources/index.html.erb]: I'm sure this is fine as is but we have a table style you might want to consider: https://www.moneyadviceservice.org.uk/en/styleguide/components/common#section-scrollable-table

Comment (urfolomeus) [inline app/views/admin/sources/index.html.erb]: Thanks, Ben. Looking into it now.

**Comment by benbarnett** [comment]
if table seems fine then :+1:

**Comment by jonalport** [comment]
:+1:

**Pull request merged**